README.md

```diff
@@ -12,9 +12,17 @@
 dos2unix build.sh
 ```
 
-Rename the `.env.example` to `.env` and set your tokens provided by magento at the [admin panel](https://commercemarketplace.adobe.com/customer/accessKeys) in the `COMPOSER_AUTH` environment variable. You can also set the version of Magento you wish to install.
+**Configure NGROK**
+In order to be able to create an account or login to an existing Doofinder account during the module initial setup, you will have to expose your local webserver to internet (to receive a callback).
+
+To do so, you can use, for example; the utility Ngrok: https://dashboard.ngrok.com/get-started/setup
```

Comment:
FYI, we have also a custom ngrok that uses a *.doofinder.com subdomain: https://github.com/doofinder/dfcommon/wiki/Doongrok

Reply:
Sure!
But as this is a public repo I didn't want to mention private tools

Reply:
Good point! Then forget about what I said before :face_in_clouds: